deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -68,23 +68,22 @@ def upload(options, screenshots)
 
     def collect_screenshots(options)
       return [] if options[:skip_screenshots]
-      return collect_screenshots_for_languages(options[:screenshots_path])
+      return collect_screenshots_for_languages(options, options[:screenshots_path])
     end
 
-    def collect_screenshots_for_languages(path)
+    def collect_screenshots_for_languages(options, path)
```

Comment:
If we are going to keep the option of skipping unsupported languages, maybe this param could be just `skip_unsupported_languages` and that can be passed to `Loader.language_folders` on line 82.